migrations/20210514174506_view_nft_token_transfers.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE view_nft_token_transfers (
+    id BIGSERIAL,
```

Comment:
Does it make sense to store the `view_nft_tokens` row ID here in this table?

Reply:
This id is an auto-incrementing id of the `view_nft_token_transfers`, it has nothing to do with `view_nft_tokens`.

Reply:
No, I mean just something like a foreign key. Though it increases complexity.